README.md

```diff
@@ -163,20 +163,41 @@ ClickstreamAnalytics.updateConfigure({
 
 ## How to integrate and test locally
 
-**Integrate**
+### Integrate by `.tgz` file
```

Comment:
Below are some rephrasing suggestions,

```markdown

## How to integrate and test locally

### Integrate the `.tgz` file

Clone this repository locally and execute the following script to generate `aws-clickstream-web-0.2.0.tgz` zip package, which will be located in the project root folder.
```bash
cd clickstream-web && npm i && npm run pack
```

Copy the `aws-clickstream-web-0.2.0.tgz` into your project, then execute the script in your project root folder to install the SDK.
```bash
npm install ./aws-clickstream-web-0.2.0.tgz
```
**Note**: Please correct the SDK version and change the path to where the `aws-clickstream-web-0.2.0.tgz` file is located.

### Integrate the `clickstream-web.min.js` file
Execute the following script to generate `clickstream-web.min.js`, located in the `/dist` folder.
```bash
cd clickstream-web && npm i && npm run pack
```
Copy the `clickstream-web.min.js` into your project and add the following initial code into your `index.html`.

```html
<script src="clickstream-web.min.js"></script>
<script>
    window.ClickstreamAnalytics.init({
        appId: 'your appId',
        endpoint: 'https://example.com/collect',
        isLogEvents: true,
        pageType: window.PageType.SPA, //multiPageApp
        sendMode: window.SendMode.Batch, //Immediate
    })
</script>
```
You can also find the `clickstream-web.min.js` file in the [Release](https://github.com/awslabs/clickstream-web/releases) page.

```

Reply:
fixed for these rephrasing suggestions.